tests/describe_task_queue_test.go

```diff
@@ -32,7 +37,7 @@ func TestDescribeTaskQueueSuite(t *testing.T) {
 	suite.Run(t, new(DescribeTaskQueueSuite))
 }
 
-func (s *DescribeTaskQueueSuite) TestNonRootLegacy() {
+func (s *DescribeTaskQueueSuite) TestNonRoot() {
```

Comment:
test this with ReportStats? we should probably make it error or not include stats, and test whichever one we choose

Reply:
+1 we don't want to support stats for a targeted partition. 

We don't want to even support DescribeTaskQueue as a whole for a single partition, we have DescribeTaskQueuePartition API now (for internal use). But `tctl` still calls this API for individual partitions.

Probably erroring in case of non-root that is asking for stats is good.

Reply:
done 👍